dali/util/nvml.h

```diff
@@ -252,6 +252,20 @@ inline bool HasCuda11NvmlFunctions() {
   return nvmlHasCuda11NvmlFunctions();
 }
 
+inline float GetDriverVersion() {
```

Comment:
I don't think this function should be inline. I know we don't have a translation unit for it - but maybe we should?

Reply:
I'd suggest splitting this function in the following way:
```
float GetDriverVersionImpl() {
  // do all the stuff that's here
}

inline float GetDriverVersion() {
  static float version = GetDriverVersionImpl();  // this is called once
  return version;
}
```

Reply:
I was thinking about that, I guess we won't be changing driver version at runtime, will we?